src/main/java/sirius/kernel/health/Exceptions.java

```diff
@@ -144,6 +147,18 @@ public ErrorHandler withNLSKey(String key) {
             return this;
         }
 
+        /**
+         * Directly specifies the message for the exception.
+         * <p>
+         * This will neither perform any NLS lookups nor append or prepend any text to the given message.
+         *
+         * @param message the message to use for the generated exception
+         * @return <tt>this</tt> in order to fluently call more methods on this handler
+         */
+        public ErrorHandler withDirectMessage(String message) {
+            return withNLSKey(MESSAGE_MODE_RAW).set(PARAM_RAW_MESSAGE, message);
+        }
```

Comment:
Ist der Gedanke nicht _eigentlich_, dass wir genau das nicht machen, und stattdessen einen NLS-Key anlegen? Wenn ichs richtig verstehe, geht diese Nachricht ja direkt an den User raus, oder? Es ist zwar manchmal ultra praktisch, sich nicht auch noch darum kümmern zu müssen, aber in Fällen von "Sollte eigentlich nie auftreten" kann man ja immer noch `withSystemErrorMessage` verwenden.

.. wobei ich in S2 auch schon einen NLS Key gesehen hab, der genau das macht, was diese Methode hier macht.

Reply:
Right, this is a special occasion where e.g. an upfront system like an ERP specifies the exact error message to show...